client/src/api.js

```diff
@@ -55,7 +55,8 @@ module.exports.signin = function (email, password) {
     // TODO: handle bad creds properly
     return post('/login', {email, password})
         .then( (response) => {
-           return response && response.user
+            // I don't think this code is reachable
```

Comment:
you right, might have been something in my env, when I had a console.log there it didn't trigger but now it seems ok